src/components/InfoTip/InfoTip.js

```diff
@@ -44,24 +44,19 @@ class InfoTip extends React.Component {
       default:
         return null;
     }
-  }
+  };
 
-  handleBackFocus() {
-    if (this.state.open) {
-      this.setState({ open: false });
-    }
-  }
-
-  handleClick(event) {
+  handleClick = event => {
```

Comment:
I though we standardized on using the bindMethods call? I don't mind this (actually prefer it) but I thought the decision was made to use the helper.

Reply:
Ah was unaware of a standardization.  I am cool to switch back if you want. Either way is essentially the same thing. The only benefit to this is typing and it matches [spec (stage-3)](https://github.com/tc39/proposal-class-fields).  Let me know thoughts on if I should switch it back.

Reply:
We should probably remain consistent here and update all instances if/when we decide to go that route (I would support that)

Reply:
Sounds good.  Opened up an issue to track that change: #294